UI/obs-app.cpp

```diff
@@ -551,7 +551,9 @@ static bool MakeUserDirs()
 		return false;
 	if (!do_mkdir(path))
 		return false;
+#endif
 
+#if defined(_WIN32) || WHATSNEW_ENABLED
```

Comment:
Can you link me to another place with this syntax? I believe @PatTheMav prefers `defined(WHATSNEW_ENABLED)` over just dropping the raw definition in place.

Reply:
This was a mistake, it was just using `#ifdef ..` before and I forgot the second `defined()`